site/docs/contribute/modules.md

```diff
@@ -83,7 +86,10 @@ This is a link to a the Helm chart stored in the [image registry]( https://helm.
 
 ```
 spec:
-  chart: "<helm chart link>" # e.g.: ghcr.io/username/chartname:chartversion
+  chart: 
+    name: "<helm chart link>" # e.g.: ghcr.io/username/
```

Comment:
This example seems wrong (needs chart name and version)

Reply:
Fixed